### DISCUSSION TITLE
How do I disable scrollbar for Shiny output?

### DISCUSSION THREAD

**Discussion by federicazoe** [quarto-dev/quarto-cli#5839] [Q&A] [answered]
### Description

I want to disable the scrollbar around a Shiny plot. That is, I want the plot to be shown fully, without needing to scroll. I searched online, on Quarto manuals and through these discussions but I was unable to find a method that worked.

Here is a reproducible example:

````
---
title: "An interactive plot that doesn't fully show :("
format: 
  html:
    page-layout: full
server: shiny
---

```{r setup}
#| context: setup
#| include: false
#| message: false

# Load libraries
library(dplyr)
library(ggplot2)
```

```{r}
selectInput(
  'var_y', label = 'Y-axis:',
  choices = colnames(iris)[1:4], 
  selected = colnames(iris)[1]
)
selectInput(
  'var_x', label = 'X-axis:',
  choices = colnames(iris)[1:4], 
  selected = colnames(iris)[2]
)
plotOutput("Scatterplot")
```

```{r}
#| context: server
#| class: myoutput

selectedData <- reactive({
    iris %>% 
    rename(x = input$var_x,
           y = input$var_y)
  })


output$Scatterplot <- renderPlot(
  { selectedData() %>% 
      ggplot() +
      aes(x = x, y = y) +
      geom_point() +
      facet_wrap(Species ~ ., ncol = 1)},
  height = 800
)

```
````

This is a screenshot of the content when I render it:

![Screenshot 2023-06-06 at 4 20 29 PM](https://github.com/quarto-dev/quarto-cli/assets/56598125/bd6f4468-e52e-4b78-a530-ab9ed1a547d2)

This is what I would like, instead:

![Screenshot 2023-06-06 at 4 42 37 PM](https://github.com/quarto-dev/quarto-cli/assets/56598125/124992fe-0216-4fb8-a3dd-52c181f33d8d)

**Comment by cderv** [comment, marked as answer]
You need to adapt the default for shiny plot output. See function parameters
https://shiny.posit.co/r/reference/shiny/1.7.4/plotoutput

It would be good to maybe try respecting some document sizing options, but this is still open issue: https://github.com/rstudio/shiny/issues/1833

**Reply by federicazoe** [reply]
Thank you! Setting `height = "800px"` within `plotOutput()` gave me indeed what I wanted. It would be useful to include an example of a plot that needs adapting shiny plot output defaults to the quarto documentation for shiny (e.g. here https://quarto.org/docs/interactive/shiny/).
